File: Listas de Exercícios (Python)/Python/Lista 2 - Sala/4.py
print("Triângulo\n")

print("--Valores decimais devem ser digitados com (.)!--")

A = float(input("Digite o lado A do triângulo: "))
B = float(input("Digite o lado B do triângulo: "))
C = float(input("Digite o lado C do triângulo: "))

print("")

if (A<B+C and B<C+A and C<B+A):
    print("Esses lados formam um triângulo!")
    if (A==B and B==C):
        print("Esse triângulo é Equilátero!")
    elif (A!=B and B!=C and C!=A):
        print("Esse triângulo é Escaleno!")
    elif (A==B or B==C or C==A):
        print("Esse triângulo é Isósceles!")
else:
    print("Esses lados não formam um triângulo!")

print("")

File: Listas de Exercícios (C)/C/Lista 2 - Sala/4.c
#include <stdio.h>

int main()
{
    float A, B, C;
    
    printf("Triângulo\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Digite o lado A do triângulo: "); scanf("%f", &A);
    printf("Digite o lado B do triângulo: "); scanf("%f", &B);
    printf("Digite o lado C do triângulo: "); scanf("%f", &C);
    
    printf("\n");
    
    if (A<B+C && B<C+A && C<B+A) {
        printf("Esses lados formam um triângulo!\n");
        if (A==B && B==C) {
            printf("Esse triângulo é Equilátero!");
        }
        else if (A!=B && B!=C && C!=A) {
            printf("Esse triângulo é Escaleno!");
        }
        else if (A==B || B==C || C==A) {
            printf("Esse triângulo é Isósceles!");
        }
    }
    else {
        printf("Esses lados não formam um triângulo!");
    }
}